tests/integration/CMakeLists.txt

```diff
@@ -0,0 +1,61 @@
+# SPDX-License-Identifier: GPL-2.0
+# Copyright (c) 2023 Meta Platforms, Inc. and affiliates.
+
+include(ExternalProject)
+
+set(BF_TOOLS_NFT_GIT_TAG v1.0.9)
+set(BF_TOOLS_NFT_SOURCE_DIR ${CMAKE_BINARY_DIR}/tools/nftables)
+set(BF_TOOLS_NFT_INSTALL_DIR ${CMAKE_BINARY_DIR}/tools/install)
+
+ExternalProject_Add(nftables
+    GIT_REPOSITORY https://git.netfilter.org/nftables
+    GIT_TAG        ${BF_TOOLS_NFT_GIT_TAG}
+
+    DEPENDS libbpfilter
+    BUILD_IN_SOURCE true
+    EXCLUDE_FROM_ALL true
+    SOURCE_DIR  ${BF_TOOLS_NFT_SOURCE_DIR}
+    INSTALL_DIR ${BF_TOOLS_NFT_INSTALL_DIR}
+
+    PATCH_COMMAND git reset --hard ${BF_TOOLS_NFT_GIT_TAG}
+        COMMAND git apply ${CMAKE_CURRENT_SOURCE_DIR}/nftables/0001-libnftables-add-support-for-bpfilter.patch
+    CONFIGURE_COMMAND ./autogen.sh
+        COMMAND
+            ${CMAKE_COMMAND} -E env
+                LDFLAGS=-Wl,-rpath=${CMAKE_BINARY_DIR}/lib\ $<$<CONFIG:Debug>:-fsanitize=address\ -fsanitize=undefined>
+                CFLAGS=-I${CMAKE_SOURCE_DIR}/lib/include\ -I${CMAKE_SOURCE_DIR}/shared/include\ $<$<CONFIG:Debug>:-fsanitize=address\ -fsanitize=undefined>
+                BPFILTER_LIBS=-L${CMAKE_BINARY_DIR}/lib\ -lbpfilter
+                BPFILTER_CFLAGS=""
```

Comment:
Is this clobbering the flags or just appending to them?

Reply:
Appending. I've modified the nftables patch to properly propagate `BPFILTER_*` flags, so `LDFLAGS` and `CFLAGS` only define `-fsanitize=*` when needed.